README.md

```diff
@@ -176,12 +176,12 @@ Put as Create
 * From [mightyroser](https://github.com/mightyroser)
 * Accessed Feb 16 2021
 
-CORS
-* From [zaidfazil](https://stackoverflow.com/questions/44037474/cors-error-while-consuming-calling-rest-api-with-react/44037631#44037631)
-* Accessed Feb 16 2021
-
```

Comment:
In case anyone is confused about why this is being deleted: this exists in lines 139-141.
Probably from a merge.